content/id/docs/tasks/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Tugas (Tasks)
+main_menu: true
+weight: 50
+content_template: templates/concept
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+
+Bagian dokumentasi Kubernetes ini berisi halaman-halaman yang perlihatkan 
+bagaimana melakukan setiap tugas (_task_). Halaman tugas menunjukkan cara melakukan
+satu hal saja, biasanya dengan memberikan urutan langkah pendek.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Antarmuka Pengguna Berbasis Web (Dasbor)
```

Comment:
```suggestion
## Antarmuka Pengguna Berbasis Web (Dashboard)
```
Revert back, after discussion with @phanama for other PR, found out that Dashboard here refer to https://github.com/kubernetes/dashboard 
so keep it as is, no need to translate to dasbor